src/workflow/OSWorkflow.cpp

```diff
@@ -368,6 +371,8 @@ spec.loader.exec_module(module)
         }
       }
     }  // End for (const auto& step : modelSteps)
+
+    workspace_->save(runDir / "in.idf");
```

Comment:
I can't tell. Is workspace in scope here? I think the EnergyPlus translator is only run if there is an EnergyPlus Measure.